src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/index.html.twig

```diff
@@ -25,22 +25,11 @@
 
 {% extends 'PrestaShopBundle:Admin:layout.html.twig' %}
 
-{% set layoutHeaderToolbarBtn = {
-  add_manufacturer: {
-    href: path('admin_manufacturers_create'),
-    desc: 'Add new brand'|trans({}, 'Admin.Catalog.Feature'),
-    icon: 'add_circle_outline',
-  },
-  add_manufacturer_address: {
-    href: path('admin_manufacturer_addresses_create'),
-    desc: 'Add new brand address'|trans({}, 'Admin.Catalog.Feature'),
-    icon: 'add_circle_outline',
-  },
-}
-%}
+
```

Comment:
Why did you remove this part?

Reply:
Damn, that's the result of a wrong manipulation on this file on my locale...  This is fixed now.